doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -664,13 +665,17 @@ Modem libraries
 
 * :ref:`lte_lc_readme` library:
 
+  * Added the function :c:func:`lte_lc_edrx_get` for reading eDRX parameters currently provided by the network.
+
   * Updated:
 
     * The functions :c:func:`lte_lc_rai_req` and :c:func:`lte_lc_rai_param_set` and the Kconfig option :kconfig:option:`CONFIG_LTE_RAI_REQ_VALUE` are now deprecated.
       The application uses the Kconfig option :kconfig:option:`CONFIG_LTE_RAI_REQ` and ``SO_RAI_*`` socket options instead.
     * The CE level enum names for :c:enum:`lte_lc_ce_level` to not include the number of repetitions.
     * The default network mode from :kconfig:option:`CONFIG_LTE_NETWORK_MODE_LTE_M` to :kconfig:option:`CONFIG_LTE_NETWORK_MODE_LTE_M_GPS`.
 
+  * Fixed a memory leak in +CEDRXS AT notification parser.
```

Comment:
I think this memory leak is worth mentioning. The devices using LTE LC can be expected to run for years, so even a small memory leak can cause the device to eventually run out of memory.

As far as I know, there are four different subsections, "Added", "Updated", "Fixed" and "Removed. And because there's only one entry for "Fixed", there's no heading for the subsection.

Reply:
> I think this memory leak is worth mentioning. The devices using LTE LC can be expected to run for years, so even a small memory leak can cause the device to eventually run out of memory.

Yep, up to you.

> As far as I know, there are four different subsections, "Added", "Updated", "Fixed" and "Removed. And because there's only one entry for "Fixed", there's no heading for the subsection.

Fine as long as @peknis is fine but this is not what is all over the document which is why I comment :) But tech writers will clean this up anyway before the release to whatever is their practice.

Reply:
Bad changelog entries slip in every now and then and we clean the changelog just before the actual release.